cypress/selectors/Login/index.jsx

```diff
@@ -0,0 +1,10 @@
+const selectors = {
```

Comment:
You would import the selectorHelpers into the selectors.js and then have that form your read and write selectors that you'll export out into your React files or tests files. We often colocate the selectors next to the components themselves to make it easier and then in the page object import that selector into the cypress folder